WordPressVIPMinimum/Sniffs/Functions/StripTagsNotAllowedSniff.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * WordPressVIPMinimum Coding Standard.
+ *
+ * @package VIPCS\WordPressVIPMinimum
+ */
+
+namespace WordPressVIPMinimum\Sniffs\Functions;
+
+use PHP_CodeSniffer_File as File;
+use PHP_CodeSniffer_Tokens as Tokens;
+
+/**
+ * This sniff will check for `strip_slashes()` usage and return a relavant error
+ */
+class StripTagsNotAllowedSniff implements \PHP_CodeSniffer_Sniff {
+
+	/**
+	 * Tokens of the whole file.
+	 *
+	 * @var array
+	 */
+	private $_tokens = array();
+
+	/**
+	 * The phpcs file.
+	 *
+	 * @var File
+	 */
+	private $_phpcsFile;
+
+
+	/**
+	 * Returns the token types that this sniff is interested in.
+	 *
+	 * @return array(int)
+	 */
+	public function register() {
+
+		return Tokens::$functionNameTokens;
+
+	}
+
+
+	/**
+	 * Processes the tokens that this sniff is interested in.
+	 *
+	 * @param \PHP_CodeSniffer\Files\File $phpcsFile The file where the token was found.
+	 * @param int                         $stackPtr  The position in the stack where
+	 *                                               the token was found.
+	 *
+	 * @return bool
+	 */
+	public function process( File $phpcsFile, $stackPtr ) {
+
+		$this->_tokens    = $phpcsFile->getTokens();
+		$this->_phpcsFile = $phpcsFile;
+
+		if ( 'strip_tags' !== $this->_tokens[ $stackPtr ]['content'] ) {
+			// We only care for strip_slashes function.
+			return false;
+		}
+
+		if ( ! $this->isFunctionCall( $stackPtr ) ) {
+			// Not a function call.
+			return false;
+		}
+
+		if ( $phpcsFile->findNext( T_COMMA, $stackPtr + 1, null, false, null, true ) ) {
```

Comment:
Instead of looking for commas, use [`$this->getMethodParameters(...)`](https://github.com/squizlabs/PHP_CodeSniffer/blob/1e440f47e304e640217645e843cdacaacc5339b6/src/Files/File.php#L1285), and then do a count.

Reply:
OK, this is the wrong advice.

`getMethodParameters()` is for getting info about the parameter - whether it's nullable, if it has a type declaration, what the variable name is, whether it is passed by reference, etc.

Much better would be to use the function parameters logic from WPCS. See https://github.com/Automattic/VIP-Coding-Standards/pull/231#issuecomment-430545956 for the links and consequent usage.